docs/content/en/_index.html

```diff
@@ -11,18 +11,18 @@
                 <span class="hero__label">
                     <span>
                         <span>
-                        Tetragon is a <b>sub-project under</b>
-                    </span>
-                    <img src="/svgs/logos/cillium.svg" width="20" height="20" loading="eager" alt="" />
-                    <span>
+                            Tetragon is a <b>sub-project under</b>
```

Comment:
ah sorry you have the huge whitespace change here as well. Maybe you want to lint this doc once for all, what would you prefer?

Reply:
My bad, I'll fix it right away. I'll just turn off auto linting for this project on my end. 

Also, would it be okay if I fix a few minor spacing issues in the hero section for mobile in this PR as well?

Reply:
yeah you can write separate commits for that, no worries, or group them, as you wish! :)